cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -991,6 +991,9 @@ class HydratorPlusPlusConfigStore {
       this.$state.go('hydrator.create', this.$stateParams, {notify: false});
     }
     let config = this.getState();
+    if (config.__ui__.nodes.length === 0) {
```

Comment:
@tonybach I was hoping we could completely eliminate this. Default to always render the dag based on config.stages. Can we do that? Is there a scenario where we require this?